resources/templates/fk_template.hh

```diff
@@ -225,14 +226,14 @@ struct {{name}}
         return true;
     }
 
-    static inline auto eefk(const std::array<float, {{n_q}}> &x) noexcept -> Eigen::Isometry3f
+    static inline auto eefk(const std::array<float, {{n_q}}> &x) noexcept -> std::array<Eigen::Isometry3f, {{num_end_effectors}}>
```

Comment:
Also probably out of scope, but we may also want an overload that takes an output array to fill as a parameter.